vortex-api/src/main/java/mil/army/usace/hec/vortex/temporal/StaticIntervalTree.java

```diff
@@ -0,0 +1,113 @@
+package mil.army.usace.hec.vortex.temporal;
+
+import java.util.*;
+
+final class StaticIntervalTree<T extends Interval> {
+    private final Node root;
+
+    /* Constructor */
+    StaticIntervalTree(List<T> intervals) {
```

Comment:
I usually pass args as an array so that type can be enforced.

Reply:
https://github.com/HydrologicEngineeringCenter/Vortex/pull/116#discussion_r1791894175